tests/wallet/test_wallet_user_store.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import aiosqlite
+import pytest
+
+from chia.util.db_wrapper import DBWrapper
+from chia.wallet.util.wallet_types import WalletType
+
+from chia.wallet.wallet_user_store import WalletUserStore
+
+
+@pytest.mark.asyncio
+async def test_store(self):
```

Comment:
```suggestion
async def test_store():
```

Leftover form the class.